chcemvediet/locale/sk/LC_MESSAGES/django.po

```diff
@@ -53,6 +53,21 @@ msgstr "Súhlasím s podmienkami používania."
 msgid "accounts:SettingsForm:anonymize_inforequests:label"
 msgstr "Zverejnené infožiadosti zverejniť anonymizovane"
 
+msgid "accounts:SettingsForm:custom_anonymization:help_text"
+msgstr "Meno, priezvisko a adresa používateľa sa anonymizujú automaticky."
+
+msgid "accounts:SettingsForm:custom_anonymization:label"
+msgstr "Pridanie položiek pre anonymizáciu"
+
+msgid "accounts:SettingsForm:custom_anonymized_strings:Error"
+msgstr "Slovné spojenie, ktoré sa anonymizuje, musí mať minimálne 3 znaky."
```

Comment:
@martinmacko47 Da sa to spravit tak, aby tu bola prenasana konstanta `WORD_SIZE_MIN`?

Reply:
`WORD_SIZE_MIN` posli ako parameter prekladoveho stringu. Vid napriklad `inforequests:obligee_action:InputBasics:delivered_date:error:older_than_previous` ako sa do prekladovych stringov posielaju parametre.